### ISSUE TITLE
This keeps on happening even after I've uninstalled and redownloaded the app

### ISSUE THREAD

**Issue by 890622** [ahkohd/switch-desktop#95]
```
TypeError: Cannot read property 'replace' of undefined
    at Lt (/Applications/Switch.app/Contents/Resources/app.asar/main.prod.js:2:36340)
    at /Applications/Switch.app/Contents/Resources/app.asar/main.prod.js:2:42573
    at Array.map (<anonymous>)
    at Xt.buildHotAppsMenuItems (/Applications/Switch.app/Contents/Resources/app.asar/main.prod.js:2:42547)
    at Xt.getContextMenuTemplate (/Applications/Switch.app/Contents/Resources/app.asar/main.prod.js:2:38323)
    at Xt.refreshTray (/Applications/Switch.app/Contents/Resources/app.asar/main.prod.js:2:42392)
    at Object.<anonymous> (/Applications/Switch.app/Contents/Resources/app.asar/main.prod.js:2:37545)
    at Object.emit (events.js:315:20)
```

---

Switch 0.5.1
Electron 11.1.1
darwin 20.3.0
Locale: en-GB

**Comment by ahkohd** [comment]
@890622 Thanks for reporting this issue. I'll release a patch that will fix this soon.

**Comment by ahkohd** [comment]
@890622 I have fixed this issue, `v0.5.2` [See the release log.](https://www.notion.so/Switch-Release-Notes-a09998f90ea04efb8f9eec1d49392c6c)
